### PR TITLE
Add LoaderConfig plugin to the Reference Viewer

### PR DESCRIPTION
**Pull request by ejeschke**
The ``LoaderConfig`` plugin allows you to configure the file openers that can be used to load various content into Ginga.
 
Registered file openers are associated with file MIME types, and there can be several openers for a single MIME type.  A priority associated with a MIME type/opener pairing determines which opener will be used for each type--the lowest priority value will determine which opener will be used.  If there are more than one opener with the same low priority then the user will be prompted for which opener to use, when opening a file in Ginga.  This plugin can be used to set the opener preferences and save it to the user's $HOME/.ginga configuration area.

Documentation added and change log updated.

### CODE REVIEW

**Comment by ejeschke** [comment]
No problem. Most users will not notice or need unless they have both `astropy` and `fitsio` installed.

**Comment by pllim** [comment]
Hmm where did the ReadTheDocs check go?

**Comment by pllim** [comment]
Oh, I misunderstood the instruction. I think we actually have to rename the YAML file.

File: ginga/rv/plugins/LoaderConfig.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+can be used to load various content into Ginga.
+
+Registered file openers are associated with file MIME types, and there can
+be several openers for a single MIME type.  A priority associated
+with a MIME type/opener pairing determines which opener will be used
+for each type--the lowest priority value will determine which opener will
+be used.  If there are more than one opener with the same low priority
+then the user will be prompted for which opener to use, when opening a
+file in Ginga.  This plugin can be used to set the opener preferences
+and save it to the user's $HOME/.ginga configuration area.
```
Comment (pllim) [inline ginga/rv/plugins/LoaderConfig.py:14]: Would be nice to give an example exactly what the filename should be in `$HOME/.ginga` and some example content of that file.

**Review by pllim — COMMENTED** [review]
What is the behavior for files not recognized by MIME, like ASDF?

**Comment by ejeschke** [comment]
> What is the behavior for files not recognized by MIME, like ASDF?

There is a set of [known types](https://github.com/ejeschke/ginga/blob/main/ginga/util/iohelper.py#L48C1-L64C1) that will be recognized if they are failed to be identified by ASDF.  The file extension ".asdf" is one of them.  But we also have a special check for ASDF embedded in FITS (especially for STScI)

File: ginga/rv/plugins/LoaderConfig.py [hunk 2]
```diff
@@ -0,0 +1,173 @@
+# This is open-source software licensed under a BSD license.
+# Please see the file LICENSE.txt for details.
+"""
+The ``LoaderConfig`` plugin allows you to configure the file openers that
+can be used to load various content into Ginga.
+
+Registered file openers are associated with file MIME types, and there can
+be several openers for a single MIME type.  A priority associated
+with a MIME type/opener pairing determines which opener will be used
+for each type--the lowest priority value will determine which opener will
```
[the remark this answers is not in the document]
Comment (ejeschke) [reply ginga/rv/plugins/LoaderConfig.py:10]: Well, it depends on how you interpret a priority number. There are systems for where a lower number has higher priority (e.g. Unix process priorities). And it does match the typical English usage, to wit: "my first priority is ...; my second priority is ...".

I think as long as the documentation is clear on this (see conspicuous note in plugin docstring) there should be no misunderstanding.

**Review by pllim — APPROVED** [review]
LGTM in principle. FFTM. Thanks!

**Comment by ejeschke** [comment]
Thanks, @pllim!

**Pull request merged**